src/testUtils/index.ts

```diff
@@ -10,3 +13,16 @@ export async function fakePromise(): Promise<void> {
   await new Promise(resolve => setImmediate(() => resolve(null)));
   await new Promise(resolve => setImmediate(() => resolve(null)));
 }
+
+/**
+ * @hidden
+ *
+ * Awaits multiple promises and advances jest time after each one
+ */
+export async function manyFakePromises(): Promise<void> {
```

Comment:
```suggestion
export async function fakePromises(): Promise<void> {
```
Just a suggestion, but you can leave this as it as well.